### DISCUSSION TITLE
Hard Mode

### DISCUSSION THREAD

**Discussion by Chris-Duncan96** [davsan56/OSRSGuesser#2] [Ideas]
What do you think of a hard mode, were instead of minimizing the photo it gets entirely removed after the delay?

**Comment by Chris-Duncan96** [comment]
5k/5k every day but one so far btw

**Reply by davsan56** [reply]
Nice job! I am looking to add more pictures in the future that should hopefully be harder 🙂

**Comment by davsan56** [comment]
Yes I have definitely thought of more game modes, I have a list of them to add eventually. I'll add this idea to that list as well!
